R/bfpackcommon.R

```diff
@@ -366,7 +293,7 @@
   # regression
   } else if (type %in%  c("regression", "regressionLogistic")) {
 
-    dependent <- decodeColNames(options[["dependent"]])
+    dependent <- decodeColNames(unlist(options[["dependent"]]))
```

Comment:
Do we still need this `decodeColNames()`? I thought all of the options decoding is handled automatically. Am I missing something @vandenman ?

Reply:
I dont know what you mean, but I need the actual names to display in a qml element from an Rsource. So I think we can just leave as is

Reply:
oh, I see now